contents/docs/user-guides/experimentation.mdx

```diff
@@ -0,0 +1,162 @@
+
+## How does it work?
+
+With Experimentation, you start with a hypothesis on how to improve your product (e.g. changing the flow, changing copy, changing visual styles, ...) and a goal (e.g. improve conversion, improve sign ups, ...). Once you have those you can:
+
+1. Create an Experiment which will have a control group and some test variants (at least one, at most three).
+
+2. Select which users will participate in the Experiment (e.g. paid users, users in a given country, or all users).
+
+Based on your user selection and your goal, we will estimate the number of users exposed and the number of days required for your Experiment to reach significant results.
+
+We distribute all selected users equally between the control and test groups. So, for example, if there are 3 test variants and 1 control variant, each variant gets 25% of the total users selected.
+
+You can tweak your settings to speed up the experiment or adjust user exposure (e.g. you may not want certain users exposed to your experiment).
+
+The actual experiment and what your user sees is controlled in your own code (see below on [Creating an experiment](#creating-an-experiment)). You ask PostHog which group does this user belong to and then you show any relevant changes in your app or website. Because this is done in your code you can test any number of things, from changing copy in a button, to taking the user to a different page, or showing them a completely different experience.
+
+Once your experiment is running, we'll automatically track your main conversion and let you know which variant leads to the best results. While the experiment is running, we'll also tell you the probability that any given variant is the winning variant.
+
+To ensure your experiment does not introduce any regressions, you can also track secondary metrics. This is particularly useful if you're optimizing a metric that has a natural trade-off with another metric (e.g. you can maximize conversion by reducing your price but then your total revenue may be reduced).
+
+Once your experiment is terminated, you can easily archive and share your results with other team members.
+
+### Results significance
+
+For your results and conclusions to be valid, any experiment must have a significant exposure. For instance, if you test a product change and only one user sees the change, you can't extrapolate from that single user that the change will be beneficial/detrimental for your entire user base. This is true for any experiment that is a simple randomized controlled experiment (e.g. this is also done when testing new drugs or vaccines).
+
+Furthermore, even if there's a lot of exposure, if, say, all variants are exposed to 10,000 participants, if the difference in conversion rate between the variants is less than 1%, it's hard to say whether one variant is truly better than the other. To be significant, there must be enough difference between the conversion rates.
```

Comment:
rereading, realised I was a bit factually incorrect here, so amending. (Got to be careful of smalll inaccuracies like these, specially in docs!)

```suggestion
Furthermore, even if there's a lot of exposure, if, say, all variants are exposed to 10,000 participants, if the difference in conversion rate between the variants is less than 1%, it's hard to say whether one variant is truly better than the other. To be significant, there must be enough difference between the conversion rates, given the exposure size.
```